.drone.starlark

```diff
@@ -0,0 +1,1073 @@
+config = {
+	'app': 'files_antivirus',
+	'rocketchat': {
+		'channel': 'builds',
+		'from_secret': 'private_rocketchat'
+	},
+
+	'branches': [
+		'master'
+	],
+
+	'appInstallCommand': 'make',
+
+	'codestyle': {
+		'ordinary' : {
+			'phpVersions': [
+				'7.0',
+				'7.1',
+				'7.2',
+				'7.3',
+			],
+		},
+	},
+
+	'phpunit': {
+		'allDatabases' : {
+			'phpVersions': [
+				'7.0',
+			]
+		},
+		'reducedDatabases' : {
+			'phpVersions': [
+				'7.1',
+				'7.2',
+				'7.3',
+			],
+			'databases': [
+				'sqlite',
+				'mariadb:10.2',
+			],
+			'coverage': False
+		},
+	},
+
+	'acceptance': {
+		'webUI': {
+			'suites': {
+				'webUIActivityList': 'webUIActList',
+			},
+			'browsers': [
+				'chrome',
+				'firefox'
+			],
+			'extraApps': [
+				'activity'
+			],
+		},
+		'api': {
+			'suites': [
+				'apiAntivirus'
+			],
+			'databases': [
+				'mariadb:10.2',
+				'mysql:5.5',
+				'mysql:5.7',
+				'postgres:9.4',
+				'oracle'
+			],
+		},
+		'api-objectstore': {
+			'suites': {
+				'apiAntivirus': 'apiAntivirusObjectstore',
+			},
+			'databases': [
+				'mariadb:10.2',
+			],
+			'extraApps': {
+				'files_primary_s3': 'composer install'
+			},
+			'extraSetup': {
+				'name': 'configure-app',
+				'image': 'owncloudci/php:7.0',
+				'pull': 'always',
+				'commands': [
```

Comment:
For testing with objectstore we need to both:
1) setup the scality config for files_primary_s3, and;
2) setup the "usual" antivirus settings